test/smoke/testApps/AgentExceptionAndDependency/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SampleTestDependencyAndExceptionWithAgent.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.applicationinsights.smoketest;
+
+import com.microsoft.applicationinsights.internal.schemav2.RemoteDependencyData;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+@UseAgent(value="test")
+@WithDependencyContainers(@DependencyContainer(value="redis", portMapping="6379"))
+public class SampleTestDependencyAndExceptionWithAgent extends AiSmokeTest {
+
+	@Test
+	@TargetUri("/trackData")
+	public void testDependencyAndExceptionWithAgent() throws Exception {
+		assertEquals(2, mockedIngestion.getCountForType("RemoteDependencyData"));
+		assertEquals(1, mockedIngestion.getCountForType("ExceptionData"));
```

Comment:
What part of the code throws this exception. Can you please point out.

Reply:
Oh, when running "jedis.set()", it will throw an exception.
Failed message ""java.net.UnknownHostException: '\/brave_joliot'",". 
About this, I talked with @littleaj  last time. It seems some code(line 227) don't run in "AiDockerClient.java"

Reply:
OK, I will take care of this with @littleaj .